lib/amazon-pricing/definitions/ec2-instance-type.rb

```diff
@@ -255,16 +255,25 @@ def self.get_name(instance_type, api_name, is_reserved = false)
       'db.m4.4xlarge' => :high,
       'db.m4.large' => :moderate,
       'db.m4.xlarge' => :high,
+      'db.m4.16xlarge' => :twentyfive_gigabit,
```

Comment:
can you update m4.16xlarge to :twentyfive_gigabit as well?

Reply:
done